.travis.yml

```diff
@@ -5,7 +5,7 @@ matrix:
   include:
   - jdk: oraclejdk8
     env: TRIPLEA_RELEASE=true
-  - jdk: oraclejdk9
+  - jdk: oraclejdk11
```

Comment:
A thing we might want to consider:
As of java 11 the Oracle JDK sort of lost its "de-facto"-JDK status, so we might want to switch to the openJDK due to the licensing restrictions.

Reply:
Second note, matching the version use for installer would be pretty slick.